src/Command/CodeStudio/CodeStudioCommandTrait.php

```diff
@@ -207,11 +207,16 @@ protected function determineGitLabProject(ApplicationResponse $cloud_application
     }
     // Prompt to create project.
     else {
-      $this->io->writeln([
+      $error_message = [
         "",
-        "Could not find any existing Code Studio project for Acquia Cloud Platform application <comment>{$cloud_application->name}</comment>.",
-        "Searched for UUID <comment>{$cloud_application->uuid}</comment> in project descriptions.",
-      ]);
+        "Could not find any existing Code Studio project for Acquia Cloud Platform application {$cloud_application->name}.",
+        "Searched for UUID {$cloud_application->uuid} in project descriptions.",
+      ];
+      if (!$create_new_cs_project) {
+        $this->io->error($error_message);
+        throw new AcquiaCliException("Contact an account manager for further assistance");
```

Comment:
Please change this exception to be meaningful out of context. i.e. "Could not find any existing Code Studio project, and the option to create a new project was not selected".

In fact you probably want to just put the `$error_message` here.

It's good to also say `Contact an account manager for further assistance` but that should not be part of the exception.

Make sure to also use the string substitutions in AcquiaCliException so that PII (the Cloud application name and UUID) are excluded.